src/modules/map/components/openlayers/utils/useDragBoxSelect.composable.js

```diff
@@ -20,16 +28,63 @@ export function useDragBoxSelect() {
     )
     dragBoxSelect.on('boxend', () => {
         const selectExtent = dragBoxSelect.getGeometry()?.getExtent()
+        if (selectExtent?.length !== 4) return
 
-        if (selectExtent?.length === 4) {
-            store.dispatch('click', {
-                clickInfo: new ClickInfo({ coordinate: selectExtent }),
-                ...dispatcher,
+        const dragBox = polygon(dragBoxSelect.getGeometry()?.getCoordinates())
```

Comment:
https://turfjs.org/docs/api/polygon I think this could break the code if `getGeomtry()` doesn't return something relevant, you might want to check that it has something valid before sending it to this TurfJS helper